pkg/storage/postgres/postgres.go

```diff
@@ -0,0 +1,168 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+type Storage struct {
+	db *pgxpool.Pool
+}
+
+func New(constr string) (*Storage, error) {
+	db, err := pgxpool.Connect(context.Background(), constr)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
тут лучше добавлять контекст ошибке, так будет гораздо проще по логам искать путь к месту, где произошла ошибка
```suggestion
	if err != nil {
		return nil, fmt.Errorf("connecting to db: %v", err)
	}
```